torch/nn/parallel/distributed.py

```diff
@@ -892,8 +892,6 @@ def gather(self, outputs, output_device):
 
     def train(self, mode=True):
         super(DistributedDataParallel, self).train(mode)
-        for module in self._module_copies[1:]:
-            module.train(mode)
```

Comment:
we still need to keep train(mode) call?

Reply:
We have the call to the parent class method on L894, which does `train(mode)` on the module for this process. this call is for the rest of the module copies on this process, but there are none.